doc/rst/language/spec/domains.rst

```diff
@@ -249,8 +244,8 @@ The type of a rectangular domain literal is defined as follows:
 
 -  ``idxType`` = the type of the range expressions;
 
--  ``stridable`` = ``true`` if any of the range expressions are
-   stridable, otherwise ``false``.
+-  ``strides`` = the most specific value that accepts all the strides
```

Comment:
Suggested rewording:

the most narrow :enum:`strideKind` that can represent all ``strides`` parameters of the range expressions.